Makefile

```diff
@@ -98,8 +98,8 @@ lint: deps
 
 .PHONY: deps
 deps:
-	test -f $(BINPATH)/golangci-lint || curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(BINPATH) v1.21.0
-	test -f $(BINPATH)/shellcheck || curl -sfL "https://storage.googleapis.com/shellcheck/shellcheck-v0.4.7.linux.x86_64.tar.xz" | tar -xJv --strip-components=1 -C $(BINPATH)
+	test -f $(BINPATH)/golangci-lint || curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/v1.27.0/install.sh | sh -s -- -b $(BINPATH) v1.27.0
+	test -f $(BINPATH)/shellcheck || curl -sfL "https://github.com/koalaman/shellcheck/releases/download/v0.7.1/shellcheck-v0.7.1.linux.x86_64.tar.xz" | tar -xJv --strip-components=1 -C $(BINPATH)
```

Comment:
this is not mentioned in the commit message, was this supposed to be a part of the same commit?

Reply:
yeah, it was intentional. I can split it out into a separate PR if you prefer.

Reply:
just an updated commit message would be good enough for me and my text search on git history :p

Reply:
I splitted it into a separate commit with a separate commit message under the same PR.